cg/models/tomte/tomte.py

```diff
@@ -41,6 +46,25 @@ def list(cls) -> list[str]:
         return list(map(lambda header: header.value, cls))
 
 
+class TomteParameters(WorkflowParameters):
+    """Model for Tomte parameters."""
+
+    gene_panel_clinical_filter: Path
+    tissue: str
+    genome: str
+
+    @validator("tissue", pre=True)
+    def restrict_tissue_values(cls, tissue: str) -> str:
+        return replace_non_alphanumeric(string=tissue)
+
+    @validator("genome", pre=True)
+    def restrict_genome_values(cls, genome: str) -> str:
+        if genome == GenomeVersion.hg38:
+            return "GRCh38"
+        elif genome == GenomeVersion.hg19:
+            return "GRCh37"
```

Comment:
We should use the genome version directly

```suggestion
    genome: GenomeVersion

    @validator("tissue", pre=True)
    def restrict_tissue_values(cls, tissue: str) -> str:
        return replace_non_alphanumeric(string=tissue)
```

Reply:
I need the validator for the genome. The values to use  in the pipeline are different than the ones used in cg:
hg19/hg38 vs GRCh37/GRCh38

Reply:
Oh I see 😓